docs/developer-docs/multi-chain/bitcoin/using-btc/btc-dev-workflow.mdx

```diff
@@ -19,16 +19,24 @@ Developers on ICP can create canisters and dapps that interact directly with the
 
 To interact with the Bitcoin testnet or mainnet networks, a developer needs to use both the management canister and Bitcoin integration API to complete the following steps:
 
-1. [Generate a Bitcoin address](generate-addresses.mdx): To submit transactions to the Bitcoin network, first your canister must generate a Bitcoin address using the `ecdsa_public_key` API endpoint of the [management canister](/docs/current/developer-docs/smart-contracts/advanced-features/management-canister#signing-and-submitting-bitcoin-transactions).
+1. [Generate a Bitcoin address](generate-addresses.mdx): To receive BTC, your canister must generate a Bitcoin address using the `ecdsa_public_key` API endpoint of the [management canister](/docs/current/developer-docs/smart-contracts/advanced-features/management-canister#signing-and-submitting-bitcoin-transactions).
 
-2. [Sign a transaction](sign-transactions.mdx): Next, a transaction must be signed using the `sign_with_ecdsa` API endpoint of the [management canister](/docs/current/developer-docs/smart-contracts/advanced-features/management-canister#signing-and-submitting-bitcoin-transactions) and an ECDSA public key. Transactions cannot be sent to the Bitcoin network until they are signed.
+2. Create a Bitcoin transaction: If the canister controls BTC, it can create a transaction to spend them.
+
+2. [Sign a transaction](sign-transactions.mdx): Next, the transaction must be signed using the `sign_with_ecdsa` API endpoint of the [management canister](/docs/current/developer-docs/smart-contracts/advanced-features/management-canister#signing-and-submitting-bitcoin-transactions). Transactions cannot be sent to the Bitcoin network until they are signed.
 
 3. [Submit a transaction](submit-transactions.mdx): Once a transaction has been signed, it can be submitted to the Bitcoin network to be executed using the Bitcoin API's [`bitcoin_send_transaction`](/docs/current/references/ic-interface-spec#ic-bitcoin_send_transaction) endpoint, which must specify a `blob` of a Bitcoin transaction and a Bitcoin network (mainnet or testnet).
```

Comment:
```suggestion
4. [Submit a transaction](submit-transactions.mdx): Once a transaction has been signed, it can be submitted to the Bitcoin network to be executed using the Bitcoin API's [`bitcoin_send_transaction`](/docs/current/references/ic-interface-spec#ic-bitcoin_send_transaction) endpoint, which must specify a `blob` of a Bitcoin transaction and a Bitcoin network (mainnet or testnet).
```